ci/release-image/docker-bake.hcl

```
@@ -45,12 +45,12 @@ function "gen_tags_for_docker_and_ghcr" {
     )
 }
 
-target "code-server-debian-11" {
+target "code-server-debian-12" {
     dockerfile = "ci/release-image/Dockerfile"
     tags = concat(
         gen_tags_for_docker_and_ghcr(""),
         gen_tags_for_docker_and_ghcr("debian"),
-        gen_tags_for_docker_and_ghcr("bullseye"),
+        gen_tags_for_docker_and_ghcr("bookwork"),
```

Oh I think we have a typo:
```suggestion
        gen_tags_for_docker_and_ghcr("bookworm"),
```